src/devices/ovrheadset/OVRHeadset.cpp

```diff
@@ -151,19 +155,167 @@ yarp::dev::OVRHeadset::OVRHeadset() :
         userPoseEnabled(false),
         logoEnabled(true),
         crosshairsEnabled(true),
-        batteryEnabled(true)
+        batteryEnabled(true),
+        inputStateError(false)
 {
     yTrace();
-}
 
+}
```

Comment:
This change is not useful, please put the bracket back in its place. Also the other spaces removed later are not necessary, but I won't block the PR for this 🙈